ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPNativeScreenViewController.swift

```diff
@@ -178,38 +178,20 @@ class FocusGuideDebugView: UIView {
         }
         return imageView
     }
-    
-    func getStyle(newStyle: SPNativeStyle?) -> Constants.UI.StandartStyle {
-        if (newStyle != nil) {
-            var style = Constants.UI.StandartStyle()
-            let newStyle = newStyle!
-            style.backgroundColor = newStyle.backgroundColor ?? style.backgroundColor
-            style.activeBackgroundColor = newStyle.activeBackgroundColor ?? style.activeBackgroundColor
-            style.onFocusTextColor = newStyle.onFocusTextColor ?? style.onFocusTextColor
-            style.onUnfocusTextColor = newStyle.onUnfocusTextColor ?? style.onUnfocusTextColor
-            style.onFocusBackgroundColor = newStyle.onFocusBackgroundColor ?? style.onFocusBackgroundColor
-            style.onUnfocusBackgroundColor = newStyle.onUnfocusBackgroundColor ?? style.onUnfocusBackgroundColor
-            style.font = newStyle.font ?? style.font
-            style.activeFont = newStyle.activeFont ?? style.activeFont
-            return style
-        } else {
-            return Constants.UI.StandartStyle()
-        }
-    }
-    
+
     @nonobjc
     @discardableResult
     func loadButton(forComponentId id: String, button: SPAppleTVButton) -> UIButton {
         if let action = components.first(where: { $0.id == id }) as? SPNativeButton {
-            let style = getStyle(newStyle: action.settings.style)
+            let style = action.settings.style
             button.isHidden = false
             button.setTitle(action.settings.text, for: .normal)
-            button.setTitleColor(UIColor(hexString: style.onUnfocusTextColor), for: .normal)
-            button.setTitleColor(UIColor(hexString: style.onFocusTextColor), for: .focused)
-            button.backgroundColor = UIColor(hexString: style.onUnfocusBackgroundColor)
+            button.setTitleColor(UIColor(hexString: style?.onUnfocusTextColor), for: .normal)
```

Comment:
It seems that `style` is now an optional, can we adopt the same strategy of providing a default at the time of decoding and remove its optionally?